web/src/beta/lib/reearth-ui/components/Radio/index.tsx

```diff
@@ -0,0 +1,80 @@
+import { FC } from "react";
+
+import { styled } from "@reearth/services/theme";
+
+export type RadioProps = {
+  value?: string;
+  label?: string;
+  disabled?: boolean;
+  checked?: boolean;
+  onChange?: (value: string) => void;
+};
+
+export const Radio: FC<RadioProps> = ({ value, label, disabled, checked, onChange }) => {
+  const handleChange = () => {
+    if (disabled) return;
+    onChange?.(value || "");
+  };
+
+  return (
+    <RadioWrapper>
+      <RadioInput
+        type="radio"
+        value={value}
+        checked={checked}
+        disabled={disabled}
+        onChange={handleChange}
+      />
+      <RadioButton checked={checked} disabled={disabled} onClick={handleChange}>
+        {checked && !disabled && <RadioIndicator checked={checked} />}
+      </RadioButton>
+      {label && <RadioLabel disabled={disabled}>{label}</RadioLabel>}
```

Comment:
I think it's better to add onClick={handleChange} to RadioLabel as well, OR move it from RadioButton to wrapper since it the cursor is pointer when mouse over the label but actually not clickable for now.